<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisControlWidget</class>
 <widget class="QWidget" name="AxisControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>258</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="stretchFactorDoubleSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.200000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="scaleTypeLabel">
       <property name="text">
        <string>Scale type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="yHighMarginLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Y high margin</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="yLowMarginLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Stretch</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Y low margin</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Background</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="AnyComboBox" name="backgroundStyleComboBox" native="true"/>
       </item>
       <item>
        <widget class="QToolButton" name="backgroundTextureChoiceToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Background Color</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="ColorChoiseWidget" name="backgroundColorChoiceWidget" native="true"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Y label</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="AnyComboBox" name="scaleTypeComboBox" native="true"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Legend title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="LineEditWithUnicodeDialog" name="yLabelLineEdit" native="true"/>
     </item>
     <item row="5" column="1">
      <widget class="LineEditWithUnicodeDialog" name="legendTitleLineEdit" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnyComboBox</class>
   <extends>QWidget</extends>
   <header location="global">commonWidgets/anycombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorChoiseWidget</class>
   <extends>QWidget</extends>
   <header location="global">commonWidgets/colorchoisewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineEditWithUnicodeDialog</class>
   <extends>QWidget</extends>
   <header>commonWidgets/lineeditwithunicodedialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
